### DISCUSSION TITLE
jsconfig.json can't find a file

### DISCUSSION THREAD

**Discussion by niklasrammelkamp** [neuefische/questions#503] [Web] [answered]
Hi there, 
my `jsconfig.json` is missing a file.
It can't find the `pages/api/comments/index.js` and that's because it doesn't exist on purpose. I don't need/want this file.

Here is a Screenshot of what's the Problem:

<img width="788" alt="Bildschirm­foto 2023-02-06 um 15 31 52" src="https://user-images.githubusercontent.com/118187307/216998550-1f2c1a87-625b-4349-93cf-3adeaa2a59ee.png">

Thanks for help 🙏

**Comment by niklasrammelkamp** [comment, marked as answer]
It's gone on its own. But maybe one of you has clue where it should come from.

**Reply by doemser** [reply]
Sounds like a little hiccup, the file must have been there once and then you deleted it, didn't you?

**Reply by niklasrammelkamp** [reply]
I would say I never created it but I'm not a hundred percent sure.
But you're probably right about the hiccup
